apps/aechannel/src/aesc_offchain_state.erl

```diff
@@ -179,19 +278,20 @@ tx_round(Tx) ->
     {Mod, TxI} = aetx:specialize_callback(Tx),
     Mod:round(TxI).
 
-fallback_to_stable_state(State) ->
-    [_|_] = drop_until_mutually_signed(State).
+-spec fallback_to_stable_state(state()) -> state().
+%% TODO: handle empty state?
```

Comment:
The empty state should hardly be hit, since we couldn't really fall back past the initial funding state?